test/up.sh

```diff
@@ -0,0 +1,194 @@
+                   -l "app=conjur-oss,release=$(get_val CONJUR_RELEASE_NAME)" \
+                   -o jsonpath="{.items[0].metadata.name}")
+
+  set_val CONJUR_POD_NAME "${CONJUR_POD_NAME}"
+}
+
+function exec_conjur() {
+  kubectl --namespace "$(get_val TEST_NAMESPACE)" \
+   exec -i "$(get_val CONJUR_POD_NAME)" --container=conjur-oss \
+    -- "$@"
+}
+
+function register_conjur_client_pod() {
+  local CONJUR_CLIENT_POD_NAME="conjur-client-pod"
+
+  # start the CLI pod
+  kubectl --namespace "$(get_val TEST_NAMESPACE)" run "${CONJUR_CLIENT_POD_NAME}" \
+   --restart='Never' \
+   --image cyberark/conjur-cli:5 \
+   --command -- sleep infinity
+
+  # wait for CLI pod to be ready
+  kubectl --namespace "$(get_val TEST_NAMESPACE)" \
+   wait --for=condition=ready "pod/${CONJUR_CLIENT_POD_NAME}" --timeout 150s
+
+  # login to conjur
+  kubectl --namespace "$(get_val TEST_NAMESPACE)" \
+    exec -i "${CONJUR_CLIENT_POD_NAME}" -- \
+     bash -xce "
+yes yes | conjur init -u '$(get_val CONJUR_URL)' -a '$(get_val CONJUR_ACCOUNT)'
+
+# API key here is the key that creation of the account provided you in step #2
+conjur authn login -u admin -p '$(get_val CONJUR_ADMIN_API_KEY)'
+
+# Check that you are identified as the admin user
+conjur authn whoami
+"
+
+  set_val CONJUR_CLIENT_POD_NAME "${CONJUR_CLIENT_POD_NAME}"
+}
+
+function exec_conjur_client() {
+  kubectl --namespace "$(get_val TEST_NAMESPACE)" \
+   exec -i "$(get_val CONJUR_CLIENT_POD_NAME)" \
+    -- "$@"
+}
+
+function register_conjur_admin_key() {
+  local CONJUR_ADMIN_API_KEY=$(exec_conjur \
+   conjurctl role retrieve-key "$(get_val CONJUR_ACCOUNT):user:admin")
+
+  set_val CONJUR_ADMIN_API_KEY "${CONJUR_ADMIN_API_KEY}"
+}
+
+function setup_conjur() {
+  exec_conjur conjurctl account create "$(get_val CONJUR_ACCOUNT)"
+}
+
+function run_policy() {
+  set_val "policy" "$(./policy.yml.sh)"
+  get_val "policy" | exec_conjur_client conjur policy load --replace root /dev/stdin
+
+  exec_conjur_client bash -xce "
+# Generate OpenSSL private key
+openssl genrsa -out ca.key 2048
+
+CONFIG='
+[ req ]
+distinguished_name = dn
+x509_extensions = v3_ca
+[ dn ]
+[ v3_ca ]
+basicConstraints = critical,CA:TRUE
+subjectKeyIdentifier   = hash
+authorityKeyIdentifier = keyid:always,issuer:always
+'
+
+# Generate root CA certificate
+openssl req -x509 -new -nodes -key ca.key -sha1 -days 3650 -set_serial 0x0 -out ca.cert \
+  -subj '/CN=conjur.authn-k8s.$(get_val AUTHENTICATOR_ID)/OU=Conjur Kubernetes CA/O=$(get_val CONJUR_ACCOUNT)' \
+  -config <(echo \"\${CONFIG}\")
+
+# Verify cert
+openssl x509 -in ca.cert -text -noout
+
+# Set cert values
+conjur variable values add conjur/authn-k8s/$(get_val AUTHENTICATOR_ID)/ca/key \"\$(cat ca.key)\"
+conjur variable values add conjur/authn-k8s/$(get_val AUTHENTICATOR_ID)/ca/cert \"\$(cat ca.cert)\"
+"
+}
+
+function populate_variables() {
+    exec_conjur_client conjur variable values add test-app-secrets/username 'meow meow meow'
```

Comment:
I'm curious why this constant `meow meow meow` isn't in the store

Reply:
I changed this so that we set this at runtime